tests/ui/pages/main/main-page.ts

```diff
@@ -0,0 +1,25 @@
+import BasePage from '../base-page';
+import InputField from "../../elements/input-field";
+
+const SELECTORS = {
+    searchBar: `.MuiBox-root.css-18l5dsu div input`,
```

Comment:
can we get rid of technical info like `.MuiBox-root.css-18l5dsu` from selectors in this and other files?